client/src/templates/RouteWithSubRoutes.js

```diff
@@ -58,9 +58,9 @@ export const RouteWithSubRoutes = (route) => {
 };
 
 const mapDispatchToProps = {};
-const mapStateToProps = ({ emailVerified, isAuthenticated }) => ({
-  emailVerified,
-  isAuthenticated,
+const mapStateToProps = ({ session }) => ({
+  emailVerified: session.emailVerified,
+  isAuthenticated: session.isAuthenticated,
```

Comment:
we might need to fine tune this a bit, it's not really a "session:" and the auth & user logic is a bit confusing at the moment but we can still improve that later I guess

Reply:
Ah ok sounds good